pylintrc

```diff
@@ -47,7 +47,9 @@ load-plugins=
 # D102 Missing docstring in public method
 # D105 Missing docstring in magic method
 disable=C0111,D101,D102,D105,too-few-public-methods,too-many-instance-attributes,too-many-arguments,
- protected-access,invalid-name,redefined-outer-name,fixme
+ protected-access,invalid-name,redefined-outer-name,no-self-use,too-many-public-methods,too-many-lines,
+ no-member,access-member-before-definition,attribute-defined-outside-init,fixme
```

Comment:
I'm not thrilled about disabling some of these, but the dynamic nature of the code in `model.py` defeats many of the standard checks.